charts/traction/templates/ui/configmap.yaml

```diff
@@ -32,4 +32,9 @@ data:
   UX_COVERIMAGE_COPYRIGHT: {{ .Values.ui.ux.coverImageCopyright | quote }}
   UX_OWNER: {{ .Values.ui.ux.owner | quote }}
   UX_SIDEBAR_TITLE: {{ .Values.ui.ux.sidebarTitle | quote }}
+  UX_INFO_BANNER_MESSAGE: {{ .Values.ui.ux.infoBanner.message | quote }}
+  UX_INFO_BANNER_MESSAGE_LEVEL: {{ .Values.ui.ux.infoBanner.messageLevel | quote }}
+  UX_INFO_BANNER_SHOW_MESSAGE: {{ .Values.ui.ux.infoBanner.showMessage | quote }}
+  UX_INFO_BANNER_ENVIRONMENT_NAME: {{ .Values.ui.ux.infoBanner.environmentName | quote }}
+  UX_INFO_BANNER_SHOW_ENVIRONMENT_NAME: {{ .Values.ui.ux.infoBanner.showEnvironmentName | quote }}
```

Comment:
As far as I can tell these are unused right?

Reply:
These are placeholders for one of the [blocked issues](https://github.com/bcgov/DITP-DevOps/issues/125).

Reply:
I can take them out of this branch if you want.

Reply:
But for that issue it's just setting the `message` field to "Development Instance" etc
Don't need 2 different config settings when you can just set `message`. Unless it's intended to be 2 different UIs or something, but that's not supported in the view so yeah I'd take it out until something like that is needed. Can just use `message` for everything.

Reply:
Removed.